contents/graham_scan/code/c++/graham_scan.cpp

```diff
@@ -0,0 +1,102 @@
+#include<iostream>
+#include<vector>
+
+struct point {
+  double x;
+  double y;
+};
+
+point origin;
+
+void swap(point &p1, point &p2){
+  point temp = p1;
+  p1 = p2;
+  p2 = temp;
+}
+
+int distSq(point p1, point p2){
+  return (p1.x - p2.x)*(p1.x - p2.x) +(p1.y - p2.y)*(p1.y - p2.y);
+}
+
+int ccwCheck(point a, point b, point c){
+
+  int crossProduct = (b.y - a.y) * (c.x - b.x) - (b.x - a.x) * (c.y - b.y);
+  if (crossProduct == 0)
+  {
+    return 0;
+  }
+  return ( crossProduct > 0)? -1 : 1;
```

Comment:
It's unnececcary to return 1,0 or -1. It's not a bool, so we have to compare it again, therefore, why not just return it?
Also, the previous `if` is useless, too.
```suggestion
  return (b.y - a.y) * (c.x - b.x) - (b.x - a.x) * (c.y - b.y);
```